conn.go

```diff
@@ -789,31 +789,47 @@ func (c *Conn) WriteMessages(msgs ...Message) (int, error) {
 // operation, it either fully succeeds or fails.
 //
 // If the compression codec is not nil, the messages will be compressed.
-func (c *Conn) WriteCompressedMessages(codec CompressionCodec, msgs ...Message) (int, error) {
+func (c *Conn) WriteCompressedMessages(codec CompressionCodec, msgs ...Message) (nbytes int, err error) {
+	nbytes, _, _, _, err = c.writeCompressedMessages(codec, msgs...)
+	return
+}
+
+func (c *Conn) WriteCompressedMessagesAt(codec CompressionCodec, msgs ...Message) (nbytes int, partition int32, offset int64, appendTime time.Time, err error) {
+	return c.writeCompressedMessages(codec, msgs...)
+}
+
+func (c *Conn) writeCompressedMessages(codec CompressionCodec, msgs ...Message) (nbytes int, partition int32, offset int64, appendTime time.Time, err error) {
+
+	nbytes = 0
+	partition = -1
+	offset = -1
+	appendTime = time.Unix(-1, 0)
```

Comment:
Those special values will modify the behavior of the `WriteMessages` and `WriteCompressedMessages` methods, how about just keeping the zero values as defaults?

Reply:
Why change behavior? They are not used: `nbytes, _, _, _, err = c.writeCompressedMessages(codec, msgs...)`

Ok, I will make zero values as defaults.